engine/dlib/src/dlib/time_posix.cpp

```diff
@@ -26,8 +26,15 @@ namespace dmTime
 
     uint64_t GetTime()
     {
-        timeval tv;
-        gettimeofday(&tv, 0);
-        return ((uint64_t) tv.tv_sec) * 1000000U + tv.tv_usec;
+        struct timespec ts;
+        clock_gettime(CLOCK_REALTIME, &ts);
+        return (uint64_t)ts.tv_sec * 1000000U + ts.tv_nsec / 1000U;
+    }
+
+    uint64_t GetMonotonicTime()
+    {
+        struct timespec ts;
+        clock_gettime(CLOCK_MONOTONIC, &ts);
```

Comment:
Replaced with new function which does the same as old, but we don't need 2 includes